modules/publinkIdSystem.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * This module adds the PublinkId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/conversantIdSystem
```

Comment:
`conversantIdSystem` could be confusing, maybe you should rename it to `publinkIdSystem`.